macros/schema_tests/aggregate_functions/expect_column_stdev_to_be_greater_than.sql

```diff
@@ -0,0 +1,42 @@
+{% macro test_expect_column_stdev_to_be_greater_than(model, column_name,
+                                                    greater_than_value,
+                                                    group_by=None,
+                                                    row_condition=None
+                                                    ) -%}
+    {{ adapter.dispatch('test_expect_column_stdev_to_be_greater_than', 
+        packages = dbt_expectations._get_namespaces()) (model, column_name,
+                                                    greater_than_value,
+                                                    group_by,
+                                                    row_condition
+                                                    ) }}
+{%- endmacro %}
+
+{% macro default__test_expect_column_stdev_to_be_greater_than(model, column_name,
+                                                    greater_than_value,
+                                                    group_by,
+                                                    row_condition
+                                                 ) %}
+with calc as (
+    select distinct
```

Comment:
Would this be functionally equivalent to a simple `group by` if specified and a non-window `stdev` function? I'd prefer that to make this a bit more standard SQL. I haven't run the tests yet, but a windowed stdev is even new to something like Snowflake. Not sure this exists everywhere, and it also makes this a little harder to follow than a more standard implementation.

Reply:
You're absolutely right. I've derived this test from a more complex query. I've refactored the test to remove the window partition.